Assets/Scripts/Input/Player.cs

```diff
@@ -304,7 +306,9 @@ private IEnumerator GoalAnimation()
             else
             {
                 Run(0);
-                JumpAction();
+                transform.GetChild(0).transform.eulerAngles = new Vector3(0, lookAtCrowd, 0);
+                animator.SetBool("isDancing", true);
```

Comment:
Is currently not  working.
Also this does not need to be in the loop(had the loop only so that they would keep continuesly jumping)